toplevel/group/group.go

```diff
@@ -425,3 +427,105 @@ func dryRunOutput(instanceAddr string, groups []vault.Item, action string) {
 		}).Infof("[Dry Run] [Vault Identity] group to be %s", action)
 	}
 }
+
+// Output a list of groups and counts of users that will be affected by policy changes
+func outputPolicyAffectedGroups(desired []group) {
+	policyActions := toplevel.GetPolicies()
+
```

Comment:
nit: include a check for `len(policyActions) > 0`
avoid unnecessary iterations of `desired`

Reply:
probably will look cleaner to check for `== 0` and return if so